code/__DEFINES/mobs.dm

```diff
@@ -562,6 +562,13 @@
 #define SIGN_TRAIT_BLOCKED 3
 #define SIGN_CUFFED 4
 
+/// Height defines
+#define CARBONHEIGHT_SHORTEST "shortest height"
+#define CARBONHEIGHT_SHORT    "short height"
+#define CARBONHEIGHT_MEDIUM   "medium height"
+#define CARBONHEIGHT_TALL     "tall height"
+#define CARBONHEIGHT_TALLEST  "tallest height"
```

Comment:
No alignment indentation. https://github.com/tgstation/tgstation/blob/master/.github/guides/STYLE.md#tabs-not-spaces

Also, rename to CARBON_HEIGHT, two words.